examples/static-evaluation.tsx

```diff
@@ -1,4 +1,5 @@
 import { styled } from '@compiled/react';
+import '@compiled/react';
```

Comment:
nit: this isn't needed - css prop is enabled when the module is found so the above `@compiled/react` has enabled it 👍

Reply:
I don't know why I added this. 🤣